packages/SwingSet/test/basedir-promises-2/bootstrap.js

```diff
@@ -26,7 +26,7 @@ export function buildRootObject(vatPowers, vatParameters) {
         // const p4 = vats.left!checkHarden(p1);
         // log(`p4 frozen ${Object.isFrozen(p4)}`);
         // allP.push(p4);
-        Promise.all(allP).then(_ => {
+        return Promise.all(allP).then(_ => {
```

Comment:
yep, that's ok because test-promises.js isn't paying attention to the `bootstrap()` result promise, so it's ok if it resolves at a different crank than before